internal/pkg/print/markdown/document/document.go

```diff
@@ -46,13 +46,14 @@ func Print(document *doc.Doc, settings settings.Settings) (string, error) {
 }
 
 func getInputDefaultValue(input *doc.Input, settings settings.Settings) string {
-	var result = " -"
```

Comment:
This extra `SPACE` was intentional, to put before the value of _non aggregated types_, otherwise we will have the following situation:

    Default: `something`         <--- this is ok

    Default:<extra space here>   <--- Markdown lint will complain about this

    ````
    <json formatted>
    ````

Reply:
I have been aware of your intentions here but wasn't aware that the linter has a problem with this. To me, the idea of adding this space to the representation of an input's default value seemed misplaced. Let me think about this.

Reply:
Yes I see your point. I originally had thought about it, but figured putting an extra if/else in `printInput()` might not be the cleanest way.

Reply:
Exactly, and there are a couple of conditions to decide on " " or "", which would make the code look ugly. I will be following a different approach and implement a `func markdown.Sanitize()` that cleans the Markdown output before we write it to stdout. Will have this pushed by Monday latest.

Reply:
That's a good approach, and even can be applied to `table` as well, in case.